build-parent/pom.xml

```diff
@@ -99,7 +99,7 @@
 
     <version.maven.aether>3.2.5</version.maven.aether>
 
-    <version.resteasy>3.0.24.Final</version.resteasy>
+    <version.resteasy>3.6.2.Final</version.resteasy>
```

Comment:
Would it work with 3.6.1.Final? That's what WildFly 14.0.1.Final has.

Reply:
I'll try it out. SmallRye is on 3.6.2, that's why I used it.

Reply:
Ah OK. Dependency alignment is a nightmare :-(

Reply:
turns out it doesn't work either way, WF11 brings in 3.0.24.Final. I must've run something wrong while testing locally. I will wait with this change for WF14

Reply:
If you wish, you can try with the WF14 branch: https://github.com/sberyozkin/thorntail/commits/THORN-2207-wf14 (Just note that we regularly rebase and force-push :-) )

Reply:
I tried it on this branch and it worked, I'll rebase this PR on top of wf14 when it's merged. Thanks!